### PR TITLE
Replace client with clientset in more controllers

### PR DESCRIPTION
**Pull request by caesarxuchao**
Continuing #20312, this PR replaces unversioned client with the clientset in all controllers.

Depends on #20320.

### CODE REVIEW

**Comment by caesarxuchao** [comment]
This is ready for review, code change related to DiscoveryClient can be ignored, they are in #20320.

**Comment by caesarxuchao** [comment]
Rebased and fixed the integration test.

**Comment by caesarxuchao** [comment]
It's mostly mechanic changes. Should be easy to review.

File: pkg/client/typed/generated/extensions/unversioned/scale_expansion.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+package unversioned
+
+import (
+	"k8s.io/kubernetes/pkg/api/meta"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+)
+
+type ScaleExpansion interface {
```
Comment (lavalamp) [inline pkg/client/typed/generated/extensions/unversioned/scale_expansion.go]: Perhaps a godoc comment?

File: pkg/client/typed/generated/extensions/unversioned/scale_expansion.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	Get(string, string) (*extensions.Scale, error)
```
Comment (lavalamp) [inline pkg/client/typed/generated/extensions/unversioned/scale_expansion.go]: Name the parameters for self-documentation.

**Comment by lavalamp** [comment]
Just minor comments.

**Comment by caesarxuchao** [comment]
Thanks. I've addressed the comment and squashed. I'm going to add the label.

**Comment by caesarxuchao** [comment]
@k8s-bot e2e test this please. issue: #17873

**Comment by caesarxuchao** [comment]
http://kubekins.dls.corp.google.com:8081/job/kubernetes-pull-build-test-e2e-gce/26416/testReport/junit/(root)/Kubernetes%20e2e%20suite/Cluster_level_logging_using_Elasticsearch_should_check_that_logs_from_pods_on_all_nodes_are_ingested_into_Elasticsearch/

**Comment by caesarxuchao** [comment]
Closed in favor to #20452

**Pull request closed without merging**